fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/LoadCommand.java

```diff
@@ -0,0 +1,438 @@
+import org.apache.doris.tablefunction.ExternalFileTableValuedFunction;
+import org.apache.doris.tablefunction.HdfsTableValuedFunction;
+import org.apache.doris.tablefunction.S3TableValuedFunction;
+
+import com.google.common.annotations.VisibleForTesting;
+import com.google.common.base.VerifyException;
+import com.google.common.collect.ImmutableList;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.Set;
+import java.util.TreeMap;
+
+/**
+ * export table
+ */
+public class LoadCommand extends Command implements ForwardWithSync, Explainable {
+    public static final Logger LOG = LogManager.getLogger(LoadCommand.class);
+    private final String labelName;
+    private final BulkStorageDesc bulkStorageDesc;
+    private final List<BulkLoadDataDesc> sourceInfos;
+    private final Map<String, String> properties;
+    private final String comment;
+    private final List<LogicalPlan> plans = new ArrayList<>();
+    private Profile profile;
+
+    /**
+     * constructor of ExportCommand
+     */
+    public LoadCommand(String labelName, List<BulkLoadDataDesc> sourceInfos, BulkStorageDesc bulkStorageDesc,
+                       Map<String, String> properties, String comment) {
+        super(PlanType.LOAD_COMMAND);
+        this.labelName = Objects.requireNonNull(labelName.trim(), "labelName should not null");
+        this.sourceInfos = Objects.requireNonNull(sourceInfos, "sourceInfos should not null");
+        this.properties = Objects.requireNonNull(properties, "properties should not null");
+        this.bulkStorageDesc = Objects.requireNonNull(bulkStorageDesc, "bulkStorageDesc should not null");
+        this.comment = Objects.requireNonNull(comment, "comment should not null");
+    }
+
+    /**
+     * for test print
+     * @param ctx context
+     * @return parsed insert into plan
+     */
+    @VisibleForTesting
+    public List<LogicalPlan> parseToInsertIntoPlan(ConnectContext ctx) throws AnalysisException {
+        ctx.getState().setNereids(true);
+        List<LogicalPlan> plans = new ArrayList<>();
+        for (BulkLoadDataDesc dataDesc : sourceInfos) {
+            plans.add(completeQueryPlan(ctx, dataDesc));
+        }
+        return plans;
+    }
+
+    @Override
+    public void run(ConnectContext ctx, StmtExecutor executor) throws Exception {
+        ctx.getState().setNereids(true);
+        this.profile = new Profile("Query", ctx.getSessionVariable().enableProfile);
+        // TODO: begin txn form multi insert sql
+        profile.getSummaryProfile().setQueryBeginTime();
+        for (BulkLoadDataDesc dataDesc : sourceInfos) {
+            plans.add(new InsertIntoTableCommand(completeQueryPlan(ctx, dataDesc), Optional.of(labelName), false));
```

Comment:
why use multi times insert into? we could load more than one table in one statement? how to process atomic?

Reply:
view the todo at https://github.com/apache/doris/issues/24221

Reply:
> why use multi times insert into? we could load more than one table in one statement? how to process atomic?

Currently, in new implements, there is only one dataDesc in `sourceInfos`, we will support multi `insert into` in one transaction later.